jobs/ci-kubernetes-soak-gce-federation-test.env

```diff
@@ -20,6 +20,15 @@ USE_KUBEFED=true
 FEDERATION_UP=false
 FEDERATION_DOWN=false
 
+# Where the clusters will be created.
+E2E_ZONES=us-central1-a us-central1-b us-central1-f
```

Comment:
You might also need `FEDERATION_CLUSTERS` real soon. Since you are touching this, it might be better to add it now.